pymobiledevice3/services/dvt/instruments/condition_inducer.py

```diff
@@ -1,21 +1,27 @@
+from __future__ import annotations
+
 import logging
+from typing import TYPE_CHECKING
 
 from pymobiledevice3.exceptions import PyMobileDevice3Exception
 from pymobiledevice3.services.remote_server import MessageAux
 
+if TYPE_CHECKING:
+    from pymobiledevice3.services.dvt.dvt_secure_socket_proxy import DvtSecureSocketProxyService
```

Comment:
If you need to import it, import it - don't conditionally import it

Reply:
k